Makefile

```diff
@@ -52,13 +53,44 @@ define release
 	scp -p update.rdf $(SSH_USER)@stage.mozilla.org:$(FTP_ROOT_PATH)/$1/update.rdf
 endef
 
-release: $(XPIS)
+release: $(XPIS) archive-sources
 	@if [ -z $(SSH_USER) ]; then \
 	  echo "release target requires SSH_USER env variable to be defined."; \
 	  exit 1; \
 	fi
-	ssh $(SSH_USER)@stage.mozilla.org 'mkdir -m 755 -p $(FTP_ROOT_PATH)/{win32,linux32,linux64,mac64}'
+	ssh $(SSH_USER)@stage.mozilla.org 'mkdir -m 755 -p $(FTP_ROOT_PATH)/{win32,linux32,linux64,mac64,sources}'
 	@$(call release,win32)
 	@$(call release,linux32)
 	@$(call release,linux64)
 	@$(call release,mac64)
+	scp -p ../$(SOURCE_ZIPFILE) $(SSH_USER)@stage.mozilla.org:$(FTP_ROOT_PATH)/sources/$(SOURCE_ZIPFILE)
+  # Update the "latest sources" symbolic link
+	ssh $(SSH_USER)@stage.mozilla.org 'cd $(FTP_ROOT_PATH)/sources/ && ln -fs $(SOURCE_ZIPFILE) $(ADDON_NAME)-latest-sources.zip'
+
+# Expects to find the following directories in the same level as this one:
+#
+# ios-webkit-debug-proxy (https://github.com/google/ios-webkit-debug-proxy)
+# ios-webkit-debug-proxy-win32 (https://github.com/artygus/ios-webkit-debug-proxy-win32)
+# libimobiledevice (https://github.com/libimobiledevice/libimobiledevice)
+# libplist (https://github.com/libimobiledevice/libplist)
+# libusbmuxd (https://github.com/libimobiledevice/libusbmuxd)
+# openssl (https://github.com/openssl/openssl)
+# libxml2 (git://git.gnome.org/libxml2.git)
+# libiconv (git://git.savannah.gnu.org/libiconv.git)
+# pcre (svn://vcs.exim.org/pcre2/code/trunk)
+# zlib (http://zlib.net/)
+archive-sources:
```

Comment:
Treat this as a future feature request if you wish, but it would "neat" to store the repo and commit hash for these repos, so that anyone can end up with the same sources as you. Or maybe these should be git submodules or something? Anyway, just thinking how someone would replicate your setup.